db/db.go

```diff
@@ -2124,9 +2124,6 @@ func GetSlotVizData(latestEpoch uint64) ([]*types.SlotVizEpochs, error) {
 	res := []*types.SlotVizEpochs{}
 
 	for _, b := range blks {
-		if b.Globalparticipationrate == 1 && !b.Finalized {
-			b.Globalparticipationrate = 0
-		}
```

Comment:
I believe this may be necessary since we store the global participation rate as 1.0 per default and only later update it to the real value, which results in wrong data for a short time for new epochs. Not sure if this still happens, pls check.

Reply:
I couldn't confirm this, seems like it's initialized with 0 [here](https://github.com/gobitfly/eth2-beaconchain-explorer/blob/master/db/db.go#L779).